libraries/radwordsprocessing/editing/find-and-replace/repalce-document-elements.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Replace Text with Document Elements
+page_title: Replace Text with Document Elements
+description: Learn how to find and replace text with various elements in a document using WordsProcessing.
+slug: radwordsprocessing-editing-replace-document-elements
+tags: find,search,replace
+published: True
+position: 6
+---
+
+# Replace Text with Document Elements
+
+**RadWordsProcessing** gives you the ability to search for a string in a [RadFlowDocument]({%slug radwordsprocessing-model-radflowdocument%}) instance and replace all matches with specified blocks or inlines. 
```

Comment:
It would be helpful if you list the elements the matches can be replaced with. Or at least better explain (link) what are block and inlines. Some customers might be unaware of these terms.